### DISCUSSION TITLE
PaddleOcr-CPU版本在鲲鹏920 arm架构上运行，速度非常非常低，请问有解决办法吗？

### DISCUSSION THREAD

**Discussion by zwjwhxz** [PaddlePaddle/PaddleOCR#12995] [Q&A]
CPU信息：
Architecture:           aarch64
  CPU op-mode(s):       64-bit
  Byte Order:           Little Endian
CPU(s):                 96
  On-line CPU(s) list:  0-95
Vendor ID:              HiSilicon
  BIOS Vendor ID:       HiSilicon
  Model name:           Kunpeng-920
    BIOS Model name:    HUAWEI Kunpeng 920 5251K
    Model:              0
    Thread(s) per core: 1
    Core(s) per socket: 48
    Socket(s):          2
    Stepping:           0x1
    Frequency boost:    disabled
    CPU max MHz:        2600.0000
    CPU min MHz:        200.0000
    BogoMIPS:           200.00
    Flags:              fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma dcpop asimddp asimdfhm
Caches (sum of all):    
  L1d:                  6 MiB (96 instances)
  L1i:                  6 MiB (96 instances)
  L2:                   48 MiB (96 instances)
  L3:                   96 MiB (4 instances)
NUMA:                   
  NUMA node(s):         4
  NUMA node0 CPU(s):    0-23
  NUMA node1 CPU(s):    24-47
  NUMA node2 CPU(s):    48-71
  NUMA node3 CPU(s):    72-95

**Comment by gaoluhang** [comment]
请问解决了吗，是怎么回事。现在也是在飞腾ARM架构下推理速度很慢很慢。转换成C++用ppocr识别也是。

**Comment by wuxiutong** [comment]
我编译成c++代码测试过了aarch64平台下cpu版本，速度很慢，一张包含了200多汉字与英文混合的986*171,40kb的识别耗时：cpu到8核再往上增加内核也不提升速度了。
![image](https://github.com/user-attachments/assets/3c4fb9a3-f530-4752-b7c2-e491c636aa1f)

**Comment by wangAlvin0202** [comment]
同问，这个问题有办法解决吗？
